### PR TITLE
Release 2.1.3 -- build an arm variant image

### PR DESCRIPTION
**Pull request by briskt**
### Added
- Add an arm64 variant to the Docker image

### CODE REVIEW

**Review by forevermatt — APPROVED** [review]
😁

File: .github/workflows/build-and-publish.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Build and Publish
 
 on:
   push:
-    branches: [ 'main', 'master' ]  # Trigger on push to 'main'& 'master' master will be remove step by step
+    branches: [ 'main', 'master' ]
```
Comment (mtompset) [inline .github/workflows/build-and-publish.yml:5]: "master should be removed eventually" might be a useful comment to leave.

Comment (briskt) [reply .github/workflows/build-and-publish.yml:5]: Maybe. Or maybe it's self-evident.

**Comment by briskt** [comment]
Resolves #10

**Pull request merged**